src/pages/Authentication/Authentication.css

```diff
@@ -0,0 +1,114 @@
+.auth-container {
+  height: 90vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.auth-card {
+  width: 35%;
+  padding: 2rem;
+  margin: 0 auto;
+  box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 29px 0px;
```

Comment:
You should use variables for colours.

Reply:
This is for box shadow and I am only using it once hence didn't add in variable as it didn't make sense but thanks for the suggestion will keep in my going forward can you please approve the pr